recipes/zwatershed/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "zwatershed" %}
+{% set version = "0.5" %}
+{% set checksum = "df5b36634e38012e0fb3b2545d3bfc26" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: {{ checksum }}
+
+build:
+  skip: true  # [win or py3k]
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - boost 1.61.*
+    - python
+    - setuptools >=18.0
+    - numpy x.x
+    - numpy >=1.7
```

Comment:
You don't need this with the setting above. `conda-forge` only build `numpy >=1.10` BTW.